src/renderer/components/TimelineSpace/Contents/Cards/Toot.vue

```diff
@@ -497,19 +514,26 @@ export default {
     }
   }
 
-  .fill-line {
-    height: 1px;
-    background-color: var(--theme-border-color);
-    margin: 4px 0 0;
-  }
-
   .action-pop-over {
     color: #303133;
   }
 }
 
-.toot:focus {
+.filterd {
```

Comment:
__[Misspell]__

"filterd" is a misspelling of "filtered"


<!-- Comment by [Sider](https://sider.review) -->